assets/js/media-grid.js

```diff
@@ -36,5 +36,14 @@
         initGrid();
       }, delay);
     });
+
+
+    $('.media-mondrian').masonry({
```

Comment:
The “loading bug” happened because you start Masonry before all images are loaded... All your object have `height:0`, so he cannot set the right position. You have 2 solutions: starting Masonry on `$(window).load();` or using the width/height given by the api to set the right proportions of the images wrappers with inline styles.